src/frontend/scss/objects/_blogpost_glimpses.scss

```diff
@@ -80,6 +80,29 @@ $richie-blogpost-glimpse-foot-cta-background-hover: $firebrick6 !default;
     padding: $richie-blogpost-glimpse-list-title-padding;
     text-align: $richie-blogpost-glimpse-list-title-textalign;
   }
+
+  &__categories {
```

Comment:
You have a small problem in there. `category-plugin-tag` which are displayed inside this div are spaced with a margin-left, except for the `first-of-type`.

With wrapping elements, lines 2 and later are off to the right. This could be solved by just applying the margin on the right?

Reply:
Nicely caught :muscle: Your solution was applied.